packages/manager/core/api/src/client.ts

```diff
@@ -42,25 +43,37 @@ export const v6 = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/apiv6',
 });
-v6.interceptors.response.use(null, handleAuthenticationError);
 
 export const aapi = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/2api',
 });
-aapi.interceptors.response.use(null, handleAuthenticationError);
 
 export const ws = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/ws',
 });
-ws.interceptors.response.use(null, handleAuthenticationError);
 
 export const v2 = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/api/v2',
 });
-v2.interceptors.response.use(null, handleAuthenticationError);
 
-export const apiClient = { v6, aapi, ws, v2 };
+export const apiClient: any = { v6, aapi, ws, v2 };
+
+Object.keys(apiClient).forEach((api) => {
+  // Configure all headers with request tagger
+  apiClient[api]?.interceptors?.request.use(function(config: any) {
+    const headers = getHeaders(config?.baseURL);
+    Object.keys(headers).forEach((header) => {
+      config.headers.set(header, headers[header]);
+      return header;
+    });
+    return config;
+  });
+  // Handle all response with authentification error
+  apiClient[api]?.interceptors?.response.use(null, handleAuthenticationError);
+  return api;
+});
+
```

Comment:
Can't you just put this in the default config ?

Or at least, declare the function and apply it before declaring const apiClient = {v6, v2, aapi, ws};
```
[v6,v2,aapi,ws].forEach(axiosInstance => axiosInstance.interceptors.request.use(addRequestTagger))
```